### DISCUSSION TITLE
any examples about using AUTH_REMOTE_USER in airflow

### DISCUSSION THREAD

**Discussion by ysun-square** [apache/airflow#15620] [Q&A]
We have our own system in our company to do authentication and authorization for our user. Information is in http headers. Feel AUTH_REMOTE_USER is the direction to go. We do not want airflow login page.

thanks

**Comment by XD-DENG** [comment]
Airflow's webserver is using **Flask AppBuilder** under the hood. 

- https://airflow.apache.org/docs/apache-airflow/stable/security/webserver.html?highlight=webserver#other-methods explains how you can change auth method via making corresponding changes in `$AIRFLOW_HOME/webserver_config.py`.
- For more details, like examples, **Flask AppBuilder**'s [Security](https://flask-appbuilder.readthedocs.io/en/latest/security.html?highlight=AUTH_REMOTE_USER#authentication-methods) doc may be a better source.

**Reply by ysun-square** [reply]
thanks. I saw these docs. just did not see a detailed example about that particular auth type. 

I can figure that out. Just need to spend some time

**Reply by XD-DENG** [reply]
👍
Looking around **Flask AppBuilder**'s doc/community discussion may lead you to more/better information, given that's what's running underlying.
